Scripts/RemediationScripts/Remediate-EnableMicrosoftDefenders.ps1

```diff
@@ -0,0 +1,877 @@
+
+            Enable-AzureDefender -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -EnableAppService - EnableStorage
+        
+        To know more about parameter execute:
+            Get-Help Enable-AzureDefender -Detailed
+            
+        To roll back:
+        1. Run below command to roll back Azure Defender for subscription with all the resource type. 
+           
+            Remove-ConfigAzureDefender -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath  C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\AzureDefender\RemediatedResourceType.csv
+        
+        To know more about parameter execute:
+   
+            Get-Help Remove-ConfigAzureDefender -Detailed
+
+########################################
+#>
+function Setup-Prerequisites
+{
+    <#
+    .SYNOPSIS
+    This command would check pre requisites modules.
+    .DESCRIPTION
+    This command would check pre requisites modules to perform remediation.
+    #>
+
+    $requiredModules = @("Az.Resources", "Az.Security", "Az.Accounts")
+    
+    Write-Host "Required modules: $($requiredModules -join ', ')" -ForegroundColor $([Constants]::MessageType.Info)
+    Write-Host "Checking if the required modules are present..."
+
+    $availableModules = $(Get-Module -ListAvailable $requiredModules -ErrorAction Stop)
+
+    # Check if the required modules are installed.
+    $requiredModules | ForEach-Object {
+        if ($availableModules.Name -notcontains $_) {
+            Write-Host "Installing [$($_)] module..." -ForegroundColor $([Constants]::MessageType.Info)
+            Install-Module -Name $_ -Scope CurrentUser -Repository 'PSGallery' -ErrorAction Stop
+            Write-Host "[$($_)] module is installed." -ForegroundColor $([Constants]::MessageType.Update)
+        }
+        else {
+            Write-Host "[$($_)] module is present." -ForegroundColor $([Constants]::MessageType.Update)
+        }
+    }
+    Write-Host "All required modules are present." -ForegroundColor $([Constants]::MessageType.Update)
+}
+
+function Enable-AzureDefender
+{
+    <#
+    .SYNOPSIS
+    This command would help in remediating 'Azure_Subscription_Config_Enable_MicrosoftDefender_Databases',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_ResourceManager',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_AppService',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_Storage',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_Container',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_Servers',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_KeyVault' control.
+    
+    .DESCRIPTION
+    This command would help in remediating 'Azure_Subscription_Config_Enable_MicrosoftDefender_Databases',
+    'Azure_Subscription_Config_Enable_MicrosoftDefender_ResourceManager',
```

Comment:
same spacing issue here as well for database line

Reply:
Looks good in code